rpm/authselect.spec.in

```diff
@@ -236,8 +236,9 @@ if [ -f %{validfile} ]; then
     %__rm -f %{validfile}
 fi
 
-# Add nss-altfiles if we are on Kinoite or Silverblue
-if %__grep -Ei "kinoite|silverblue" /etc/os-release &> /dev/null; then
+# Add nss-altfiles if we are on an ostree/rpm-ostree based system.
+# Applies to Fedora IoT, Silverblue, Kinoite, CoreOS.
```

Comment:
Let's link to https://github.com/authselect/authselect/issues/48
which has all the background discussion, and I think it's less obscure if we just say "applies to rpm-ostree systems" or so.